docs/source-pytorch/cli/lightning_cli_advanced.rst

```diff
@@ -1,113 +1,169 @@
 :orphan:
 
-#######################################
-Eliminate config boilerplate (Advanced)
-#######################################
+#################################################
+Configure hyperparameters from the CLI (Advanced)
+#################################################
 **Audience:** Users looking to modularize their code for a professional project.
 
-**Pre-reqs:** You must have read :doc:`(Control it all from the CLI) <lightning_cli_intermediate>`.
+**Pre-reqs:** You must have read :doc:`(Mix models and datasets) <lightning_cli_intermediate_2>`.
+
+As a project becomes more complex, the number of configurable options becomes very large, making it inconvenient to
+control through individual command line arguments. To address this, CLIs implemented using
+:class:`~pytorch_lightning.cli.LightningCLI` always support receiving input from configuration files. The default format
+used for config files is yaml.
+
+.. tip::
+
+    If you are unfamiliar with yaml, have a look at the short introduction at `realpython.com#yaml-syntax
+    <https://realpython.com/python-yaml/#yaml-syntax>`__.
+
 
 ----
 
-***************************
-What is a yaml config file?
-***************************
```

Comment:
It was added by @williamfalcon, I would keep the example and combine it with your reference to the other tutorial.

Reply:
Done.